recipe/meta.yaml

```diff
@@ -11,16 +11,16 @@ source:
 build:
   noarch: python
   script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
-  number: 0
+  number: 1
 
 requirements:
   host:
-    - python >=3.9,<4.0
+    - python {{ python_min }}
     - poetry-core
     - pip
   run:
     - __unix  # [unix]
-    - python >=3.9,<4.0
+    - python >={{ python_min }},<4.0
```

Comment:
For future reference, pinning python `<4` isn't recommended. If there is ever a Python 4 release, it's supposed to not be breaking like the 2->3 release.